draft-ietf-shmoo-remote-fee.md

```diff
@@ -127,7 +125,8 @@ free for everyone but to always have a free option that can be used without any
 the registration procedure itself. As long as there are still enough paying participants to cover
 the base costs, additional participant can effectively be added without increasing expenses.
 
-It is not in scope for this document or the SHMOO working group to make suggestions for changing
+This document is a product of the SHMOO working group and based on the charter work of this group
+it is not in scope for this document to make suggestions for changing
```

Comment:
```suggestion
It is not in scope for this document to make suggestions for changing
```

I would suggest reverting this - I see no reason why the SHMOO charter needs to be mentioned.